content/blog/2018/06/2018-05-14-simple-pull-request-plugin.adoc

```diff
@@ -0,0 +1,166 @@
+---
+layout: post
+title: "GSoC Project Intro: Simple pull request Plugin"
```

Comment:
Simple Pull-Request Job plugin (as in text)?
Maybe it would make sense to add "define Pipeline as YAML" or so to highlight some sweet features

Reply:
Subtitle? :)